tests/test_unit_generator_docker.py

```diff
@@ -86,6 +88,24 @@ def test_prepare_content_sets_should_not_fail_when_cs_is_empty(tmpdir):
             assert generator._prepare_content_sets({}) is False
 
 
+def test_large_labels_should_break_lines(tmpdir):
+    image = Image(yaml.safe_load("""
+    from: foo
+    name: test/foo
+    version: 1.9
+    labels:
+      - name: 'the.large.label'
+        value: Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec pretium finibus lorem vitae pellentesque. Maecenas tincidunt amet.
+    """), 'foo')
+    with docker_generator(tmpdir) as generator:
+        generator.image = image
+        with cekit_config(redhat=True):
+            generator.add_build_labels()
+            for label in generator.image.labels:
+                if label.name == 'the.large.label':
```

Comment:
This is also something that is probably not needed. How about referencing the single label as:

```python
generator.image.labels[0]
```

Reply:
The `add_build_labels` function adds more labels to the list, that's why I added the loop instead of looking into the position. If the function changes in the future, might breck the test case. But I agree that is cleaner referencing the index instead. wdyt? Keep the loop?

Reply:
Well, looking again, the function is appending values to the list. Unlikely that this will break the test in the future. Changing.

Reply:
Even if it will break in the future - it's fine. This means that the contract was broken and it's good to know about it.